src/views/ModifyPropertyView.vue

```diff
@@ -0,0 +1,241 @@
+  { label: 'Laubwald', value: 'Laubwald' },
+  { label: 'Nadelwald', value: 'Nadelwald' },
+  { label: 'Mischwald', value: 'Mischwald' },
+  { label: 'Gehölz', value: 'Gehölz' },
+  { label: 'Waldfläche, noch nicht aufgeschlüsselt', value: 'Waldfläche, noch nicht aufgeschlüsselt' },
+  { label: 'Fließgewässer', value: 'Fließgewässer' },
+  { label: 'Kanal', value: 'Kanal' },
+  { label: 'Hafen', value: 'Hafen' },
+  { label: 'Bach, Graben', value: 'Bach, Graben' },
+  { label: 'Stehendes Gewässer', value: 'Stehendes Gewässer' },
+  { label: 'Sumpf', value: 'Sumpf' },
+  { label: 'Wasserfläche, noch nicht aufgeschlüsselt', value: 'Wasserfläche, noch nicht aufgeschlüsselt' },
+  { label: 'Militärisches Übungsgelände', value: 'Militärisches Übungsgelände' },
+  { label: 'Anderes Übungsgelände', value: 'Anderes Übungsgelände' },
+  { label: 'Schutzfläche', value: 'Schutzfläche' },
+  { label: 'Historische Anlage', value: 'Historische Anlage' },
+  { label: 'Friedhof', value: 'Friedhof' },
+  { label: 'Unland', value: 'Unland' },
+  { label: 'Nutzung noch nicht zugeordnet', value: 'Nutzung noch nicht zugeordnet' },
+];
+
+const originalValues = ref<{
+  title: string;
+  description: string;
+  district: string;
+  corridor: string;
+  parcel: string;
+  landRegistry: string;
+  usageType: string | null;
+}>({
+  title: '',
+  description: '',
+  district: '',
+  corridor: '',
+  parcel: '',
+  landRegistry: '',
+  usageType: null,
+});
+
+onMounted(() => {
+  if (id.value) {
+    fetchPropertyDetails();
+  }
+});
+
+const fetchPropertyDetails = () => {
+  projectService
+      .getProperty(props.projectId, id.value.toString())
+      .then((property) => {
+        title.value = property.title || '';
+        description.value = property.description || '';
+        district.value = property.district || '';
+        corridor.value = property.corridor || '';
+        parcel.value = property.parcel || '';
+        landRegistry.value = property.landRegistry || '';
+        usageType.value = property.usageType || null;
+
+
+        originalValues.value = {
+          title: title.value,
+          description: description.value,
+          district: district.value,
+          corridor: corridor.value,
+          parcel: parcel.value,
+          landRegistry: landRegistry.value,
+          usageType: usageType.value,
+        };
+      })
+      .catch((err) => {
+        console.error('Fehler beim Laden der Objektdetails:', err);
+      });
+};
+
+
+const isModified = computed(() => {
+  return (
+      title.value !== originalValues.value.title ||
+      description.value !== originalValues.value.description ||
+      district.value !== originalValues.value.district ||
+      corridor.value !== originalValues.value.corridor ||
+      parcel.value !== originalValues.value.parcel ||
+      landRegistry.value !== originalValues.value.landRegistry ||
+      usageType.value !== originalValues.value.usageType
+  );
+});
```

Comment:
this kind of implementation surely works, however, there's another way to make it less verbose. Maybe you could try this suggestion:

```suggestion
const isModified = computed(() => {
  return (Object.keys(originalValues.value) as Array<keyof typeof originalValues.value>).some(
    (key) =>
      originalValues.value[key] !== (key === 'usageType' ? usageType.value : eval(key + ".value"))
  );
});
```